hack/setup/infra/manage.kserve-helm.sh

```diff
@@ -21,16 +21,28 @@
 #
 # Environment variables:
 #   USE_LOCAL_CHARTS      - Use local charts instead of OCI registry (default: false)
-#   KSERVE_VERSION        - KServe version to install (default: from kserve-deps.env)
+#   SET_KSERVE_VERSION    - KServe version to install (default: from kserve-deps.env)
```

Comment:
not sure about it, KSERVE_VERSION sounds good to me, any specific reason to update it?